edx_repo_tools/oep2/explode_repos_yaml.py

```diff
@@ -27,11 +27,11 @@
 )
 def implode(hub, org, branch):
     """
-    Implode all openedx.yaml files, and print the results as formatted output.
+    Implode all catalog-info.yaml or openedx.yaml files, and print the results as formatted output.
     """
     data = {
         repo.full_name: openedx_yaml
         for repo, openedx_yaml
-        in iter_openedx_yaml(hub, org, branch)
+        in iter_openedx_release_yaml(hub, org, branch)
```

Comment:
Hmm, it may not make sense to update this tool at all.  This one is specifically for reading the openedx.yaml and shouldn't mix it with catalog-info.yaml.

Reply:
I fixed it.